.semgrepignore

```diff
@@ -1 +0,0 @@
-.github/workflows/
```

Comment:
@enishant , please don't delete this one.

Reply:
Included file(s) which was get deleted while git merge ours.